_posts/ja/newsletters/2021-01-13-newsletter-ja.md

```diff
@@ -0,0 +1,162 @@
+  そのサポートを[watchtower][topic watchtowers]の実装に追加し、コストを削減し安全性を向上させています。
+  このリリースでは、[PSBT][topic psbt]の作成と署名のための一般的なサポートも追加され、いくつかのBug Fixが含まれています。
+
+{% comment %}<!--
+- Bitcoin Core 0.20.2rc1 および 0.19.2rc1 はこのニュースレターの公開後、
+  いつかの時点でare expected to be[利用可能][bitcoincore.org/bin]になる予定です。
+  [Newsletter #110][news110 bcc19620]で説明されている改善などいくつかのBug Fixが含まれており、
+  理解できない将来のtaprootトランザクションの再ダウンロードを防ぎます。
+-->{% endcomment %}
+
+## 注目すべきコードとドキュメントの変更
+
+*今週の[Bitcoin Core][bitcoin core repo]、[C-Lightning][c-lightning repo]、[Eclair][eclair repo]、[LND][lnd repo]、[Rust-Lightning][rust-lightning repo]、
+[libsecp256k1][libsecp256k1 repo]、[Hardware Wallet Interface (HWI)][hwi repo]、[Rust Bitcoin][rust bitcoin repo]、
+[Bitcoin Improvement Proposals（BIP）][bips repo]、および[Lightning BOLTs][bolts repo]の注目すべき変更点。*
+
+- [Bitcoin Core #18077][]は、一般的な自動ポートフォワードプロトコル[NAT-PMP][rfc 6886]
+  (Network Address Translation Port Mapping Protocol)のサポートを導入しています。
+  `-natpmp`設定パラメータを付けて起動されたBitcoin clientは、NAT-PMP対応ルーターのリスニングポートを自動的に開きます。
+  NAT-PMPのサポートは、複数のセキュリティ問題によりBitcoin Core 0.11.1以降無効化されていた既存のUPnP (Universal Plug and Play)の
+  サポートと並行して追加されています。UPnPとは対照的に、NAT-PMPはXML解析の代わりに固定サイズのUDPパケットを使用するため、
+  [リスクが少ない][laanwj natpmp]と考えられています。この変更により、NAT-PMPの後方互換性がある後継である
+  [PCP][rfc 6887] (Port Control Protocol)のサポートが追加されます。
+
+- [Bitcoin Core #19055][]は、今後のPRで予定されている機能に使用できるよう[MuHashアルゴリズム][muhash algorithm]を追加しています。
+  [newsletter 123][muhash review club]で取り上げたように、MuHashはローリングハッシュアルゴリズムで、
+  オブジェクトのセットのハッシュダイジェストを計算し、アイテムが追加・削除された際に効率的に更新することが可能です。
+  これは、[2017年にPieter Wuilleによって][muhash mailing list]提案され、[Bitcoin Core #10434][]で実装された
+  全UTXOセットのダイジェストを計算するためにMuHashを使用するというアイディアを復活させるものです。
+  UTXOセットの統計用のインデックス作成の進捗を追跡し、[assumeUTXO][topic assumeutxo]アーカイブの検証を容易にすることに興味がある人のために、
+  メタPR[Bitcoin Core #18000][]にプロジェクトの進捗と今後の手順が記載されています。
+
+- [C-Lightning #4320][]では、`invoice`RPCに`cltv`パラメーターが追加され、ユーザーやプラグインが
+  インボイスの`min_final_cltv_expiry`フィールドを設定できるようになりました。
+
+- [C-Lightning #4303][]では、`hsmtool`が標準入力（stdin）でパスフレーズを取得し、
+  コマンドラインで指定されたパスフレーズを無視するよう更新されています。
+
+{% include references.md %}
+{% include linkers/issues.md issues="18077,19055,4320,4303,6993,20477,18000,10434" %}
+[bitcoin core 0.21.0]: https://bitcoincore.org/bin/bitcoin-core-0.21.0/
+[lnd 0.12.0-beta]: https://github.com/lightningnetwork/lnd/releases/tag/v0.12.0-beta.rc5
+[news63 bcc15759]: /en/newsletters/2019/09/11/#bitcoin-core-15759
+[daftuar disabletx]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2021-January/018340.html
+[rfc 6886]: https://tools.ietf.org/html/rfc6886
+[rfc 6887]: https://tools.ietf.org/html/rfc6887
+[laanwj natpmp]: https://github.com/bitcoin/bitcoin/issues/11902#issue-282227529
+[wuille bech32m post]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2021-January/018338.html
+[bech32m bip]: https://github.com/sipa/bips/blob/bip-bech32m/bip-bech32m.mediawiki
+[news83 podle]: /en/newsletters/2020/02/05/#podle
```

Comment:
Newsletter-83 is available in Japanese. So '/ja/' is better.

Reply:
@harding I wonder if there's some magic we can do where if page `/<lang>/X` links to page Y then:

- the link is to `/<lang>/Y` if it exists;
- otherwise the link is to `/en/Y`

That means these internal links would automatically improve if we add translations later.